### DISCUSSION TITLE
Choose config file or key from artisan

### DISCUSSION THREAD

**Discussion by glennwilton** [spatie/laravel-backup#1508] [Ideas]
Could we have the option in the command line to choose an alternative config file or key,  allowing for different types of backups.

i.e. backup:run --config=backup_user_files

You could choose either a different key in the array from the same backup config file OR a different config file.

$config = config('backup')  << Change here to choose a different php file
$backupDestinations = BackupDestinationFactory::createFromArray($config['backup']);  <<< here for a different key

My preference would be choosing an alternative config file, as that way, backup/monitor/notifications/cleanup are all changed together, cleaner to manage.

**Comment by PaolaRuby** [comment]
You could make that on your own implementation without problems
